library/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/SessionReplayConfiguration.kt

```diff
@@ -66,14 +69,28 @@ internal constructor(
             return this
         }
 
+        /**
+         * Sets the sampling rate for Session Replay recorded Sessions. Please note that this
+         * sampling rate will be applied on top of already
+         *
+         * @param samplingRate the sampling rate must be a value between 0 and 100. A value of 0
+         * means no session will be recorded, 100 means all sessions will be recorded.
+         */
+        fun sessionReplaySampleRate(@FloatRange(from = 0.0, to = 100.0) samplingRate: Float):
+            Builder {
+            this.samplingRate = samplingRate
+            return this
+        }
+
```

Comment:
we need to use term `sample rate` everywhere.

also the phrase below has unfinished sentence: it ends with `will be applied on top of already`.

```suggestion
        /**
         * Sets the sample rate for Session Replay recorded Sessions. Please note that this
         * sample rate will be applied on top of already
         *
         * @param sampleRate the sample rate must be a value between 0 and 100. A value of 0
         * means no session will be recorded, 100 means all sessions will be recorded.
         */
        fun sessionReplaySampleRate(@FloatRange(from = 0.0, to = 100.0) sampleRate: Float):
            Builder {
            this.samplingRate = sampleRate
            return this
        }

```